modules/administration/pages/ssl-certs-selfsigned.adoc

```diff
@@ -1,79 +1,25 @@
-[[custom-ssl]]
-= Custom SSL Certificates
+[[ssl-certs-selfsigned]]
+= Self-Signed SSL Certificates
 
-You can use custom certificates with {productname} and {productname} Proxy.
-
-This section covers how to use a third party SSL certificate authority with a new {productname} installation, and replacing existing certificate with new custom certificates.
-
-Before you begin, ensure you have:
-
-* A certificate authority (CA) SSL public certificate
-* An SSL server key
-* An SSL server certificate
+By default, {productname} uses a self-signed certificate.
+In this case, the certificate is created and signed by {productname}.
+This method does not use an independent certificate authority to guarantee that the details of the certificate are correct.
+Third party CAs perform checks to ensure that the information contained in the certificate is correct.
+For more on third party CAs, see xref:administration:ssl-certs-imported.adoc[].
+
+This section covers how to re-create your self-signed certificates on an existing installation.
+It also covers how to create new self-signed certificates and authenticate your existing clients to the new certificate, using an intermediate certificate.
+Intermediate certificates merge the intermediate and root CA certificates into one file.
+Ensure that the intermediate certificate comes first in the combined file.
 
 Your key and certificate files must be in PEM format.
```

Comment:
"Your key and certificate files must be in PEM format." can be removed. We generate it in the correct format already.